css/bootstrap.steamlug.css

```diff
@@ -1026,7 +1026,7 @@ p a.group-join{font-size:100%}
 @keyframes showpolls{
 	from{width:0}
 }
-.navbar-nav>li.steamLogin{right:0;padding-right:11px}
+.navbar-nav>li.steamLogin{right:0;padding-right:11px;margin-top:4px}
```

Comment:
Instead of adding a margin here, you should probably adjust the padding on the next line.

    padding:<top> <right> <bottom> <left>;

The bar is 50px high apparently judging by the rest of our CSS, so the rule on line 1030 would become this when taking your 4px adjustment into account:

    padding:7px 5px 8px 3px;